### DISCUSSION TITLE
Roles in the Github organization

### DISCUSSION THREAD

**Discussion by cypherhoodlum** [NostrGit/NostrGit#76] [General]
I'll open the discussion about the roles in the NostrGit organization. Who should be code maintainers? Who should have the rights to do what? I think right now @PeerRich holds all the strings?

**Comment by PeerRich** [comment]
correct, right now i'm the single point of failure and i shouldnt be given my full time job over at https://github.com/calcom/cal.com

**Comment by cypherhoodlum** [comment]
Happy have more rights in the Github organization :)

**Comment by PeerRich** [comment]
added you as maintainer!

**Comment by cypherhoodlum** [comment]
More people welcome, I think :)

**Comment by brycedev** [comment]
happy to maintain, if needed

**Comment by cypherhoodlum** [comment]
@PeerRich I think you need to add us as People to the organisation before we can accept the invite to be maintainers. I don't have NostrGit listed in my organisations at the moment.

**Comment by cypherhoodlum** [comment]
Nevermind, I'm just tripping. I think I have all the rights I need.

**Comment by PeerRich** [comment]
@brycedev upgraded you to maintainer! 

p.s. i cant really be active during the workdays due to my main job, will heavily work on weekends!

**Comment by cypherhoodlum** [comment]
> p.s. i cant really be active during the workdays due to my main job, will heavily work on weekends!

Understandable. We're starting to have a good amount of people, fortunately.

**Comment by pierresisson** [comment]
same, i would love to help you if you need me @PeerRich and the team 👌🏻

**Comment by cypherhoodlum** [comment]
All contributions are welcome! :heart: 
Just communicate frequently what you are working on and join the discussion on Issues :)

**Comment by pierresisson** [comment]
I will @cypherhoodlum :D

**Comment by pierresisson** [comment]
@cypherhoodlum if you need more maintainer i'm really active on this repo :)

**Reply by cypherhoodlum** [reply]
@PeerRich +1

**Comment by alltheseas** [comment]
As a product person I'm happy to put together a product board using the Github projects feature, and group feature requests into milestones. If this might be helpful for the team I think I may need some permissions for this?

**Comment by cypherhoodlum** [comment]
We [decided](https://github.com/NostrGit/NostrGit/discussions/95) to not use Discussions anymore. Let's continue the thread at #124   :heart:
